rclpy/src/rclpy/_rclpy_logging.cpp

```diff
@@ -66,12 +67,16 @@ rclpy_logging_shutdown()
  * \return None
  */
 void
-rclpy_logging_set_logger_level(const char * name, int level)
+rclpy_logging_set_logger_level(const char * name, int level, bool detailed_error)
```

Comment:
If we don't want to update https://github.com/Barry-Xu-2018/rclpy/blob/d52133d49e77228e91dfc80831c938fb9d87ae75/rclpy/rclpy/impl/rcutils_logger.py#L245, we might need to update

```suggestion
rclpy_logging_set_logger_level(const char * name, int level, bool detailed_error = false)
```

and

```
  m.def("rclpy_logging_set_logger_level",
    &rclpy_logging_set_logger_level,
    py::arg("name"), py::arg("level"), py::arg("detailed_error") = false);
```

Reply:
Yes. Avoid other caller to directly this.